documentation/en/draft_modulators.md

```diff
@@ -0,0 +1,370 @@
+    
+#### 2.2.2 Two-dimensional or surface wave 
+The two-dimensional wave propagates in two directions. They can travel in any surface, this is why they can also be called surface wave, like the waves produced in a liquid surface at rest when a rock is let drop in it.
+
+
+#### 2.2.3 Three-dimensional o esféricas
+Son ondas que se propagan en tres direcciones. Las ondas tridimensionales se conocen también como ondas esféricas, porque sus frentes de ondas son esferas concéntricas que salen de la fuente de perturbación expandiéndose en todas direcciones. El sonido es una onda tridimensional. Son ondas tridimensionales las ondas sonoras (mecánicas) y las ondas electromagnéticas.
+**********
+
+### 2.3 Particles movement
+
+#### 2.3.1  Longitudinal waves
+In this type of wave the particles vibrate parallel to the propagation of the wave. For example, the seismic waves, sound waves and compressiong an elastic spring results in a longitudinal wave. 
+
+#### 2.3.2 Transverse waves
+In the transverse waves the particles vibrate perpendicularly to the propagation of the wave. Like the sea waves, waves in a string, and seismic waves.
+
+### 2.4 Periodicity
+
+#### 2.4.1 Periodic wave
+The local disturbance that originates this type of waves it's produced in repetitive cicles, e.g. a sine wave.
+
+#### 2.4.2 Aperiodic Non-periodic wave 
+
+The disturbance that originates them are isolated, or in case it repeats, the following disturbances have different characteristics. These isolated waves are known as pulse wave.
+
+
+# Chapter 2 Modulators and demodulators
+
+Information signals are rarely in an appropiate state for transmission, so that signals must be carry between the transmisor and the receiver on some medium for transmission.
+
+Modulation is the process of transforming the information from its original form to a more appropiate one for transmission. 
+
+Demodulation is the reverse process, turning the modulated wave to its original form. Modulation its done in the transmisor, specifically in the modulator, and the demodulation in the receiver side on a circuit called demodulator or detector.
+
+## 2.1 Carrier wave
+
+A carrier wave is an electric wave that can be altered in some of its parameters for the information signal (sound, image or data) to obtain a modulated signal and be transported on a communication channel.
+
+The use of a carrier wave helps solve many circuit, antenna, propagation and noise problems. That's why a practical antenna must have an approximate size to the lenght of the electromagnetic wave to be transmitted. If soundwaves diffused directly in the form of electromagnetic signals, the antenna would have the height of more than 1 Km. By using frequencies much more higher for the carrier, the size of the antenna its reduced drastically because higher frequencies have shorter wavelenght. 
```

Comment:
```suggestion
The use of a carrier wave helps solve many circuit, antenna, propagation and noise problems. That's why a practical antenna must have an approximate size to the length of the electromagnetic wave to be transmitted. If sound waves diffused directly in the form of electromagnetic signals, the antenna would have the height of more than 1 Km. By using frequencies much more higher for the carrier, the size of the antenna its reduced drastically because higher frequencies have shorter wavelength. 
```